sources/Genesis.move

```diff
@@ -446,6 +447,9 @@ module Genesis {
             GenesisNFT::initialize(&genesis_account, merkle_root, 1639u64, image);
         };
         StdlibUpgradeScripts::do_upgrade_from_v6_to_v7_with_language_version(&genesis_account, 4);
+        StdlibUpgradeScripts::do_upgrade_from_v11_to_v12(&genesis_account);
+        // Initialize Frozen strategy
+        FrozenConfigStrategy::do_initialize(&association);
```

Comment:
Why did this function parameter use &association? I noticed that most initialize functions use &genesis_account. If you use &genesis_account, you can put FrozenConfigStrategy::do_initialize into StdlibUpgradeScripts::do_upgrade_from_v11_to_v12.

Reply:
If you put resources in the genesis account, it will be more troublesome to maintain the list. Because the ability to modify Config is placed in the Genesis account
1. If the modification ability is taken out during associal initialization, the signature of Genesis is required.
2. If you give resources to the genesis account and capabilities to the Associal account in the Genesis stage, it is feasible, but currently we need to run part of the upgrade process

Reply:
OK